src/global.d.ts

```diff
@@ -43,6 +43,6 @@ declare global {
 		/** Set by the server for whitelabel domains. */
 		whitelabelCustomizations: WhitelabelCustomizations | undefined,
 		/** Whether we are running inside dialog of desktop app. */
-		webDialog: true | undefined,
+		nativeAppWebauthn: NativeApp | undefined,
```

Comment:
I specifically wanted to disable sw for all webdialogs in the future

Reply:
my suggestion would then be to rename it to `nativeAppWebdialog`; If we ever find other reasons to use webdialogs, I would assume we would still be using this same preload script and we only really need to expose a single `nativeApp`

Reply:
I am in favor of clearly separating normal IPC and webDialog one, I am okay with `nativeAppWebdialog`, just don't want to hardcode webauthn anywhere